actionpack/lib/action_dispatch/http/filter_redirect.rb

```diff
@@ -1,16 +1,14 @@
 # frozen_string_literal: true
 
+require "action_dispatch/http/parameter_filter"
+
 module ActionDispatch
   module Http
     module FilterRedirect
       FILTERED = "[FILTERED]" # :nodoc:
 
       def filtered_location # :nodoc:
-        if location_filter_match?
-          FILTERED
-        else
-          location
-        end
+        location_filter_match
```

Comment:
I'm mistaken. The parameter filter logic can't replace this because it is doing an includes match on the location.